lnwire/allow_htlc_response.go

```diff
@@ -0,0 +1,128 @@
+package lnwire
+
+import (
+	"fmt"
+	"io"
+)
+
+type AllowHTLCStatus uint8
+
+const (
+	AllowHTLCStatus_Allow AllowHTLCStatus = 1
```

Comment:
The prefix on all the members of the enum should be removed in order to match with the existing popularized golang style.